src/main/java/org/jointheleague/discord_bot_example/Bot.java

```diff
@@ -1,106 +1,122 @@
+		}
+
+		api.getServerTextChannelsByName(channelName).forEach(e -> e.sendMessage("Bot Connected"));
+
+		// add Listeners
+
+		RandomNumber randomNumber = new RandomNumber(channelName);
+		api.addMessageCreateListener(randomNumber);
+		helpListener.addHelpEmbed(randomNumber.getHelpEmbed());
+
+		SetProfilePic setPFP = new SetProfilePic(channelName);
+		api.addMessageCreateListener(setPFP);
+		helpListener.addHelpEmbed(setPFP.getHelpEmbed());
+
+		ToGif toGif = new ToGif(channelName);
+		api.addMessageCreateListener(toGif);
+		helpListener.addHelpEmbed(toGif.getHelpEmbed());
+
+		RandomCase randomCase = new RandomCase(channelName);
+		api.addMessageCreateListener(randomCase);
+		helpListener.addHelpEmbed(randomCase.getHelpEmbed());
+
+		_ApiExampleListener apiExampleListener = new _ApiExampleListener(channelName);
+		api.addMessageCreateListener(apiExampleListener);
+		helpListener.addHelpEmbed(apiExampleListener.getHelpEmbed());
+
+		api.addMessageCreateListener(helpListener);
+		api.addMessageCreateListener(new MomBot(channelName));
+		api.addMessageCreateListener(new DadJokes(channelName));
+		api.addMessageCreateListener(new ClockMessageListener(channelName));
+		api.addMessageCreateListener(new CalculatorMessageListener(channelName));
+		api.addMessageCreateListener(new ComicMessageListener(channelName));
+		api.addMessageCreateListener(new ElmoMessageListener(channelName));
+		api.addMessageCreateListener(new FactMessageListener(channelName));
+		api.addMessageCreateListener(new CasinoGameListener(channelName));
+		api.addMessageCreateListener(new PEMDASListener(channelName));
+		api.addMessageCreateListener(new Ryland(channelName));
+		api.addMessageCreateListener(new RockPaperScissorsListener(channelName));
+		api.addMessageCreateListener(new leetMessageListener(channelName));
+		api.addMessageCreateListener(new ConnectFour(channelName));
+		api.addMessageCreateListener(new FlagMessageListener(channelName));
+		api.addMessageCreateListener(new SimpleEmbedBuilder(channelName));
+		api.addMessageCreateListener(new PictureOf(channelName));
+		api.addMessageCreateListener(new GetPicture(channelName));
+		api.addMessageCreateListener(new CuteAnimal(channelName));
+		api.addMessageCreateListener(new Weather(channelName));
+		api.addMessageCreateListener(new FashionAdvisor(channelName));
+		api.addMessageCreateListener(new MorseTranslator(channelName));
+		api.addMessageCreateListener(new HangmanListener(channelName));
+		api.addMessageCreateListener(new BogoSorterListener(channelName));
+		api.addMessageCreateListener(new ComplimentListener(channelName));
+		api.addMessageCreateListener(new FEHStatListener(channelName));
+		api.addMessageCreateListener(new CrazyEights(channelName));
+		api.addMessageCreateListener(new Blackjack(channelName));
+		api.addMessageCreateListener(new MinesweeperListener(channelName));
+		api.addMessageCreateListener(new NewPollMessageListener(channelName));
+
+		api.addReactionAddListener(new TicketSystem());

```

Comment:
TicketSystem does not exist.

Reply:
Oops, sorry, that was a new thing I was working on and then later cancelled, I don't know how that got into this branch.